src/igvfd/schemas/changelogs/publication.md

```diff
@@ -0,0 +1,3 @@
+## Changelog for publication.json
+
+### Minor changes since version 1
```

Comment:
I don't think we should put the line about minor changes if there were no changes